src/ar/tuktukcinema/src/eu/kanade/tachiyomi/animeextension/ar/tuktukcinema/Tuktukcinema.kt

```diff
@@ -293,7 +318,7 @@ class Tuktukcinema : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
     // preferred quality settings
     private fun getPrefHostUrl(preferences: SharedPreferences): String = preferences.getString(
         "default_domain",
```

Comment:
I would suggest changing the key to:
```suggestion
        "default_domain_v${BuildConfig.VERSION_CODE}",
```
This way, the url will be updated for everyone on an extension update.

Reply:
ok the old url is working also but changed it when i made the changes

Reply:
I think its better to include the extension version code, as just updating the default doesn't change it if a user has changed it themselves. With this, it's always going to update the url.